### ISSUE TITLE
pip release ?

### ISSUE THREAD

**Issue by michaelkuty** [django-oscar/django-oscar-api#38]
Depends on #37

**Comment by specialunderwear** [comment]
After the documentation is done, oscarapi can be released on pypi.

**Comment by michaelkuty** [comment]
Thanks for info, would be nice to have this package available on pypi.

**Comment by specialunderwear** [comment]
For the record, if anyone knows people that enjoy writing documentation. Please make them help us.

**Comment by bufke** [comment]
I wouldn't say I enjoy it - but I could possibly assist here. What's the plan? Is this for general project information or api docs like [this](http://www.django-rest-framework.org/topics/documenting-your-api/).

**Comment by specialunderwear** [comment]
Like this: http://django-oscar-api.readthedocs.org/en/latest/

It sits in the https://github.com/django-oscar/django-oscar-api/tree/documentation branch. If you could help fill in the stubs or correct mistakes that would be much appreciated. You can send pull requests on that branch to @maerteijn

**Comment by specialunderwear** [comment]
https://pypi.python.org/pypi/django-oscar-api/1.0.0
